cluster/genesis.go

```diff
@@ -836,10 +21,32 @@ func (client *cluster) generateGenesis() error {
 	if err != nil {
 		return err
 	}
-	var b bytes.Buffer
-	genesisTpl.Execute(&b, map[string]string{
-		"NetworkID": client.NetworkID,
-	})
+
+	newGenesis, err := genesis.GenerateGenesis(
+		genesis.GenesisOptions{
+			Network:                       "test",
```

Comment:
You're not using the `client.NetworkID` anymore. We should change it to `client.Network` and use it here.